apps/condo/domains/property/schema/Property.js

```diff
@@ -75,7 +80,10 @@ const Property = new GQLListSchema('Property', {
                     if (typeof value !== 'object') { return addFieldValidationError(`${JSON_EXPECT_OBJECT_ERROR}${fieldPath}] ${fieldPath} field type error. We expect JSON Object`) }
                     const { dv } = value
                     if (dv === 1) {
-                        // TODO(pahaz): need to checkIt! sections and so on
+                        if (!jsonMapValidator(value)){
```

Comment:
We need to add at least two tests for that (one positive and negative)

Reply:
Done